include/z64actor.h

```diff
@@ -357,4 +357,95 @@ typedef enum {
     DOORLOCK_NORMAL_SPIRIT
 } DoorLockType;
 
+typedef enum {
+    NAVI_ENEMY_GOHMA = 1,
```

Comment:
0 may be `NAVI_ENEMY_DEFAULT`, I don't think it's actually used (?) but message `0x0600` is "What's that?" (see extracted text in `assets/text/message_data.h`) and it would make sense for it to be intended for navi since all `0x06xx` message ids are navi enemies

Reply:
Added `NAVI_ENEMY_DEFAULT`.